variables.tf

```diff
@@ -4,30 +4,61 @@
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
 
-variable "subnet_id" {
-  type        = string
-  description = "The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0)."
+variable "users" {
+  description = "A map whose keys are the usernames of each database user and whose values are a map containing supported user attributes.  The only currently-supported attribute is \"role\" (string).  The only currently-supported roles are \"read_only\" and \"read_write\".  Example: { \"firstname1.lastname1\" = { \"role\" = \"read_only\" }, \"firstname2.lastname2\" = { \"role\" = \"read_write\" } }"
+  type        = map(object({ role = string }))
 }
 
 # ------------------------------------------------------------------------------
 # OPTIONAL PARAMETERS
 #
 # These parameters have reasonable defaults.
 # ------------------------------------------------------------------------------
-variable "ami_owner_account_id" {
+
+variable "assume_userservices_was_db_read_only_policy_description" {
+  default     = "Allows assumption of the WAS DB read-only role in the User Services account."
+  description = "The description of the IAM policy that allows assumption of the WAS DB read-only role in the User Services account."
   type        = string
-  description = "The ID of the AWS account that owns the Example AMI, or \"self\" if the AMI is owned by the same account as the provisioner."
-  default     = "self"
 }
 
-variable "aws_availability_zone" {
+variable "assume_userservices_was_db_read_only_policy_name" {
+  default     = "AssumeUserServicesWASDBReadOnlyRole"
+  description = "The name of the IAM policy that allows assumption of the WAS DB read-only role in the User Services account."
   type        = string
-  description = "The AWS availability zone to deploy into (e.g. a, b, c, etc.)."
-  default     = "a"
 }
 
-variable "aws_region" {
+variable "assume_userservices_was_db_read_write_policy_description" {
+  default     = "Allows assumption of the WAS DB read-write role in the User Services account."
+  description = "The description of the IAM policy that allows assumption of the WAS DB read-write role in the User Services account."
```

Comment:
This should reflect that it provides access to the the read-write role in the staging User Services account _**and**_ the read-write role in the production User Services account.